upgrade_versions.py

```diff
@@ -188,6 +193,21 @@ def _bump_release(self, pkg, log=None, user_name=None, user_email=None):
             assert user_email is not None
             rpm_bump_spec(pkg.specfile, log, user_name, user_email)
 
+    def _update_prerelease_tag(self):
+        if not self._prerelease:
+            self._prerelease = "%{nil}"
+
+        try:
```

Comment:
We don't really need the try block. `re.sub` does not raise exceptions if no strings are found, it just returns the unaltered string.